pkg/sql/opt/memo/statistics_builder.go

```diff
@@ -346,7 +346,7 @@ func (sb *statisticsBuilder) colStat(colSet opt.ColSet, e RelExpr) *props.Column
 		return sb.colStatLeaf(colSet, &relProps.Stats, &relProps.FuncDeps, relProps.NotNullCols)
 	}
 
-	panic(fmt.Sprintf("unrecognized relational expression type: %v", e.Op()))
+	panic(pgerror.NewAssertionErrorf("unrecognized relational expression type: %v", e.Op()))
```

Comment:
Here and in other places you could have used `log.Safe()`.

Reply:
Done, went through all of them and updated as necessary.

Reply:
Thanks! and sorry for the inconvenience.